### ISSUE TITLE
ATP-dependent protein-DNA complex displacement activity (GO:0061995)

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#22115]
Should this be
is_a
GO:0140658 | ATP-dependent chromatin remodeler activity

@colinlog 
@pgaudet

**Comment by colinlog** [comment]
I believe this is a term that is needed for Mot1 (human BTAF1 / TAFII170) and may also be the one for RAD54.  Basically, these are ATP-dependent SNF2 family members that do not work on the nucleosome. Rather, they act on TBP- or RAD51-DNA complexes, promoting their disassembly from DNA, but also their assembly under some circumstances.

SNF2-type motor activity
  > ATP-dependent nucleosome remodelling activity
        > SMARCA2/4, SMARCA1, INO80, P400, ...   all remodel nucleosomes
  > Non-nucleosome remodelling activity
         >RAD54
         >BTAF1

**Comment by pgaudet** [comment]
@colinlog you mean it's not a remodeler ? 
or should we add the parent?

**Comment by ValWood** [comment]
PomBase have used for
https://www.pombase.org/gene/SPBC887.14c
5' to 3' DNA helicase Pif1/Pfh1 
from https://www.pombase.org/reference/PMID:30053106
and

https://www.pombase.org/gene/SPBC23E6.02
DNA translocase Rrp2
https://www.pombase.org/reference/PMID:28552615
rrp2 also has this mapping:
GO:0140658 | ATP-dependent chromatin remodeler activity | IEA with IPR000330
is the human ortholog
HLTF (HGNC:11099) | helicase like transcription facto

**Comment by ValWood** [comment]
Pif1 appears to be both a helicase and a remodeller?

Rrp2, maybe I just need to ask for the mapping to 'remodeller' to be removed?

**Comment by colinlog** [comment]
> @colinlog you mean it's not a remodeler ?
> or should we add the parent?

Yes BTAF1 is a remodeller, but not of nucleosomes, rather it helps dismantle&remodel other DNA-protein complexes.  

In fact, the nucleosome-remodelling archetype SWI2/SNF2/SMARCA2-4 enzyme may also be involved in dbTF dissociation/stripping in vivo (GO:0061995), all the while making DNA available for all dbTFs, coTFs etc and any other DNA binding protein ((see my paper https://www.embopress.org/doi/epdf/10.1093/emboj/16.22.6772)) by sliding histone octamers along chromosomal DNA to generate nucleosome-depleted stretches of DNA at promoters, enhancers and potential loop bases. 

The SWI2/SNF2/SMARCA2/4 branch are truly DNA motors - with a 15 millisecond cycle of ATP hydrolysis - that can translocate along naked DNA. The parent 'SNF2-type motor activity ' would be nice to have for the motor-like biochemistry for this sub-class of SNF2-type enzymes. The DNA-stimulated motors like SWI2/SNF2/Brahma/SMARCA2-4 may be so powerful that any other protein-DNA complex than the nucleosome can be evicted, however they are really nucleosome-specific in that they can read histone modifications like H3K14acetylation that attracts them, or H3K4me3 that repels one of them.

Having a common parentage would reflect the common evolutionary history of the SNF2-type remodelers to include Mot1/BTAF1 and RAD54 as well as histone H2A variant insertion-removal activities like those of the SNF2 family members SWR1 and INO80.  "ATP-dependent remodelling activity" as the parental term, many chaperones fall under this, no?

PMID: 25957681 says that BTAF1 is absent from archea, I do not know for RAD54, but I think SWI2/SNF2 and SMARCA5 (ISWI) and chromodomain SNF2 family members are also missing from archea (as we knew them in 2015). Altough, the emerging relatedness of some archea (see figure 1 with this beautiful tree from wikipedia https://en.wikipedia.org/wiki/Tree_of_life_(biology)#/media/File:A_Novel_Representation_Of_The_Tree_Of_Life.png) to basal eukaryotes makes me wonder if that is true for every archea: what about TACK and Loki and Thor? Perhaps we should leave the idea of imposing a deep taxon limit and accordingly GO-rebranching for this term to the day you paint the 22 human SNF2 homologs with me, if you haven't done this yet Pascale?

In conclusion, the SNF2 enzymes are only one a branch of the Type II ATP-dependent helicase superfamily. But, they are not helicases.

GO:0061995* (ATP-dependent DNA stripping) is not a universal property of SNF2-type enzymes, as some evolved to act on nucleosomal histone variants without any permanent DNA stripping. While others space nucleosomes within arrays and accompany histone acetyltransferases at gene promoters. "DNA-protein complex remodelling" is their universal  activity, and it is not always the nucleosome/histones that is the target of this activity. 

a possible parentage for all the the SNF2-activities:


>SNF2 remodelling activity (Eukaryote only?]
>>DNA stripping
>>> of TBP [BTAF1]
>>> of RAD51 [RAD54]

>>Nucleosome remodelling [>20 enzymes, 8 in S.c]
>>> histone octamer transfer
>>> histone octamer sliding activity
>>>>nuckeosome array spacing
>>>>nucleosome depletion at cis-regulatory module (synonyms: Nucleosome depleted / free region = NFR / NDR)
>>> nucleosomal histone H2A variant insertion/removal
>>>>H2A.Z insertion/removal
>>>>H2Aw insertion --> enabling DNA methylation in plants 
>>>>macroH2A insertion?



Even BTAF1 is not solely a DNA stripper, it is one of the main chaperones for TBP proteins, one of TBP's many required binding partners to permit regulated transcription by RNA Pol I, II and III. 

Rad54 removes RAD51 and belongs to the homologous DNA recombination (super)process that also depends on Rad52 rings.

* the synonym of GO:0061995 should be its parent, as the switch from 'DNA' to 'nucleic acid' in the defintion it implies (rightly so, see spliceosome) that there are also RNA-protein complex strippers, an activity that many RNA helicases display as a side- or main-effect of separating base-paired RNA strands by translocating along one of them --  The SNF2 branch is about DNA translocation but not about unwinding double stranded DNA,however.

**Comment by colinlog** [comment]
the is a rewording of the above proposal on hierarchical organisation of chromatin remodeling 
CHROMATIN REMODELLING ACTIVITIES
Level1  DNA stripping
Level2 of TBP [BTAF1]
Level2 of RAD51 [RAD54]
Level1 Nucleosome remodeler ACTIVITY HAS_PART NUCLEOSOME BINDING [>20 enzymes, 8 in S.c]
Level2 histone octamer transfer [SMARCA2/4]
Level3 H2A-H2Bdimer diplacementt
Level3 H3-H4 tertramer
Level2 histone octamer sliding activity [SMARCA2/4, CHD FAMILY (S.P Hpr1)]
Level3 sub branch -nucleosome array spacing [ISWI TYPE SMARCA5]
Level3 sub-branch -nucleosome depletion at cis-regulatory module (synonyms: Nucleosome depleted / free region = NFR / NDR) [SMARCA2/4]
Level2 nucleosomal histone H2A variant insertion/removaL
Level3 sub branch H2A.Z insertion/removal [SWR1-INO80]
Level3 sub branch H2Aw insertion --> enabling DNA methylation in plants [YFR038 HUMAN HELLS]
Level3 sub branch macroH2A insertion?  (SNF2??? OR ITS CHAPERONE DEPOSITION SYSTEM?]

**Comment by ValWood** [comment]
resolve this
https://github.com/geneontology/go-ontology/issues/22115#issuecomment-915224604
when these terms are added.I'm still confused by the relation between the different types of protein-DNA displacement terms.

**Comment by pgaudet** [comment]
@colinlog I am closing this, I think we have now all the terms.
